panels/grafana-clickhouse-datasource/dashboards/data-analysis.json

```diff
@@ -1826,7 +1799,7 @@
             }
           },
           "queryType": "sql",
-          "rawSql": "SELECT modification_time as timestamp, concatAssumeInjective(database, '.', table) as table, rows FROM system.parts WHERE database IN ($database) AND table IN (${table})  AND $__timeFilter(modification_time) ORDER BY modification_time DESC",
+          "rawSql": "SELECT modification_time as timestamp, concatAssumeInjective(database, '.', table) as table, rows FROM system.parts parts WHERE parts.database IN ($database) AND parts.table IN (${table})  AND $__timeFilter(modification_time) ORDER BY modification_time ASC",
```

Comment:
Why is it ordered ASC instead of DESC?

Reply:
This is the out of the box dashboard provided by clickhouse datasource to support analysis of clickhouse cluster,
They made this changes as per grafana requiring long time series data to be in ASC order to display the graph.